trollhunter-unity/Assets/Scripts/Food.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Food {
+	private string seedText;
+

```

Comment:
No way to set the seed text.  Should at least add a constructor with input to initialize the class